Makefile.config.CUDA

```diff
@@ -0,0 +1,25 @@
+## Refer to http://caffe.berkeleyvision.org/installation.html
+# Contributions simplifying and improving our build system are welcome!
+
+# cuDNN acceleration switch (uncomment to build with cuDNN).
+# USE_CUDNN := 1
+
+# CUDA directory contains bin/ and lib/ directories that we need.
+USE_CUDA := 1
```

Comment:
USE_CUDA also can be specified in Makefile. This file isn't included in Makefile if the USE_CUDA in Makefile is not defined.

I also have reservations about USE_CUDA := 1. Naive user might think USE_CUDA := 0 turns it off. This gets to difference between behavior `#if USE_CUDA` vs `#ifdef USE_CUDA`.  If USE_CUDA :=0, #if sees its argument as false, whereas #ifdef sees it as true since all it cares is that it has been defined, not what it's value is. 

I'm not clear on what best practices are on these things; I usually try to to avoid compilation switches myself. Maybe you're way ahead of me.